cdk/lib/frontend.ts

```diff
@@ -18,5 +34,384 @@ export class Frontend extends GuStack {
+        stageValues: {
+          [Stage.CODE]: false,
+          [Stage.PROD]: true,
+        },
+      }),
+      threshold: 3,
+      evaluationPeriods: 2,
+      comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+      metric: new Metric({
+        metricName: "HTTPCode_Target_5XX_Count",
+        namespace: "AWS/ApplicationELB",
+        dimensionsMap: {
+          LoadBalancer: ec2App.loadBalancer.loadBalancerFullName,
+          TargetGroup: ec2App.targetGroup.targetGroupFullName,
+        },
+        statistic: "Sum",
+        period: Duration.seconds(60),
+      }),
+      snsTopicName: "reader-revenue-dev",
+    });
+
+    new GuAlarm(this, "HighELB5XXRateAlarm", {
+      app,
+      alarmName: `URGENT 9-5 - ${this.stage} support-frontend ELB is returning 5XX errors`,
+      alarmDescription:
+        "Impact - Some or all actions on support website are failing. Follow the process in https://docs.google.com/document/d/1_3El3cly9d7u_jPgTcRjLxmdG2e919zCLvmcFCLOYAk/edit",
+      actionsEnabled: this.withStageDependentValue({
+        app,
+        variableName: "NoHealthyInstancesAlarmEnabled",
+        stageValues: {
+          [Stage.CODE]: false,
+          [Stage.PROD]: true,
+        },
+      }),
+      threshold: 3,
+      evaluationPeriods: 2,
+      comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+      metric: new Metric({
+        metricName: "HTTPCode_ELB_5XX_Count",
+        namespace: "AWS/ApplicationELB",
+        dimensionsMap: {
+          LoadBalancer: ec2App.loadBalancer.loadBalancerFullName,
+        },
+        statistic: "Sum",
+        period: Duration.seconds(60),
+      }),
+      snsTopicName: "reader-revenue-dev",
+    });
+
+    new GuAlarm(this, "LatencyNotificationAlarm", {
+      app,
+      alarmName: `URGENT 9-5 - ${this.stage} support-frontend has high latency`,
+      alarmDescription:
+        "Impact - support-frontend users are seeing slow responses. Follow the process in https://docs.google.com/document/d/1_3El3cly9d7u_jPgTcRjLxmdG2e919zCLvmcFCLOYAk/edit",
+      actionsEnabled: this.withStageDependentValue({
+        app,
+        variableName: "NoHealthyInstancesAlarmEnabled",
+        stageValues: {
+          [Stage.CODE]: false,
+          [Stage.PROD]: true,
+        },
+      }),
+      threshold: 1,
+      evaluationPeriods: 2,
+      comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+      metric: new Metric({
+        metricName: "TargetResponseTime",
+        namespace: "AWS/ApplicationELB",
+        dimensionsMap: {
+          LoadBalancer: ec2App.loadBalancer.loadBalancerFullName,
+          TargetGroup: ec2App.targetGroup.targetGroupFullName,
+        },
+        statistic: "Average",
+        period: Duration.seconds(60),
+      }),
+      snsTopicName: "reader-revenue-dev",
+    });
+
+    // TODO: Do we still need this?
+    new GuAlarm(this, "CatalogLoadingFailureAlarm", {
```

Comment:
That would explain why we didn't get an alarm when the catalog was broken recently. I think we should probably fix this as it will bring the whole of subs down if we can't load the catalog.

Reply:
I've created a health card for this: https://trello.com/c/vUo5sO3R/368-fix-the-catalog-loading-failed-alarm-in-support-frontend